service/javadsl/integration-client/src/main/java/com/lightbend/lagom/javadsl/client/integration/LagomClientFactory.java

```diff
@@ -205,7 +206,8 @@ public void addStopHook(Callable<? extends CompletionStage<?>> hook) {
             }
         }, actorSystem.dispatcher());
 
-        ServiceInfo serviceInfo = new ServiceInfo(serviceName);
+        // TODO: review this. Building a kafka client shouldn't require the whole ServiceInfo, just the name.
```

Comment:
I would like to have a `ServiceName` type so that the constructor overload is not `String`-ly typed.